components/admin_console/license_settings/license_settings.jsx

```diff
@@ -605,5 +611,18 @@ export default class LicenseSettings extends React.PureComponent {
             </>
         );
     }
+
+    renewLicenseCard = () => {
+        if (isLicenseExpired(this.props.license) || isLicenseExpiring(this.props.license)) {
+            return (
+                <RenewLinkCard
+                    license={this.props.license}
+                    isLicenseExpired={isLicenseExpired(this.props.license)}
+                    totalUsers={Math.max(this.props.stats.TOTAL_USERS, 30)}
```

Comment:
Why max here to 30 users? We should show them the total users they are currently have in their system even if less than 30